base/applications/kbswitch/kbswitch.c

```diff
@@ -707,7 +715,8 @@ AddTrayIcon(HWND hwnd)
     tnid.hIcon = CreateTrayIcon(szKLID, szImeFile);
     StringCchCopy(tnid.szTip, _countof(tnid.szTip), szName);
 
-    Shell_NotifyIcon(NIM_ADD, &tnid);
+    if (!Shell_NotifyIcon(NIM_ADD, &tnid))
+        ERR("Shell_NotifyIcon(NIM_ADD) failed\n");
```

Comment:
Not sure we need to "care" (in the debug log) about the failure of this?

Reply:
It promotes quick debugging.